### PR TITLE
Bug 2038832: Add registry URL suggestions for centos stream templates

### PR DESCRIPTION
**Pull request by yaacov**
Description:
New CentOS stream8+9 images are missing registry suggestions in create vm form.

- [x] add missing stream suggestion images
- [x] use https://quay.io/organization/containerdisks repo for suggested images
- [x] hide only RHEL images when upstream 

Screenshot:
Peek 2022-01-10 11-20

### CODE REVIEW

**Comment by yaacov** [comment]
@glekner @pcbailey @metalice @avivtur @hstastna @upalatucci please review

**Comment by dominikholler** [comment]
Thanks for the PR. Please note that there is an overlap with https://bugzilla.redhat.com/show_bug.cgi?id=2031155 .

File: frontend/packages/kubevirt-plugin/src/components/form/helper/container-source-help.tsx [hunk 1]
```diff
@@ -4,8 +4,12 @@ import { isUpstream } from '@console/internal/components/utils';
 import {
   CENTOS7,
   CENTOS8,
```
Comment (gouyang) [inline frontend/packages/kubevirt-plugin/src/components/form/helper/container-source-help.tsx:6]: should we remove `CENTOS8` here as it's not in the template list anymore?

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/form/helper/container-source-help.tsx:6]: we want to be backworkd compatible 2 versions back

**Pull request merged**